test/utils_test.js

```diff
@@ -201,3 +202,19 @@ describe("buildAttachmentUrl", () => {
       .eql("http://cdn/file");
   });
 });
+
+describe("timeago", function() {
+  it("should convert a timestamp", () => {
+    expect(timeago(new Date().getTime() - 86400000)).eql("1 day ago");
+  });
+
+  it("should convert a date string", () => {
+    expect(timeago("2016-12-01T00:00:00.000Z",
+           new Date("2016-12-02T00:00:00.000Z"))).eql("1 day ago");
```

Comment:
I looked in `CollectionRecords` and maybe we don't have to support passing strings. I think we only want to support millisecond timestamps.